packages/eslint-plugin/src/rules/shorthand-property-sorting/index.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  shorthandBuckets,
+  shorthandFor,
+  kebabCase,
+  type ShorthandProperties,
+} from '@compiled/utils';
+import type { Rule, Scope } from 'eslint';
+import type { ObjectExpression, Property } from 'estree';
+
+import { isStyled, isCss, isCssMap } from '../../index';
```

Comment:
Seems like a circular dep? Can you move these to a utils file and import from there?

Reply:
@JakeLane just moved some things around, does that look better to you?